test/eradius_test_handler.erl

```diff
@@ -43,7 +43,8 @@ radius_request(#radius_request{cmd = request, attrs = Attrs} = _Req, _Nasprop, _
 	    end;
 	#{?User_Name := <<"Vendor-Dicts">>} ->
 	    case AttrsMap of
-		#{?IM_SSID := _, ?IM_Wifi_Connect_Type := _, ?IM_LI_Location := _} ->
+		#{?IM_SSID := _, ?IM_Wifi_Connect_Type := _, ?IM_LI_Location := Loc} ->
+		    ct:pal("--Location : ~p--~n", [Loc]),
```

Comment:
```suggestion
		    ct:pal("--Location : ~p--", [Loc]),
```

Reply:
I thought ct:pal didn't add a newline?
In any case, I will probably add a more complete trace with all the attributes in AttrsMap to have more info in case of errors.

Reply:
```erlang
1> ct:pal("~p",[1]), ct:pal("~p",[2]).

----------------------------------------------------
2023-01-23 14:10:34.479
1

Warning: ct_logs not started
1
----------------------------------------------------
2023-01-23 14:10:34.484
2

Warning: ct_logs not started
2ok
```
Plus, please take a look how it used in current module.

Reply:
ok